setup.py

```diff
@@ -83,6 +83,11 @@ def main():
         "packages": find_packages(),
         "package_data": {"hachoir.wx.resource": ['hachoir_wx.xrc']},
         "entry_points": ENTRY_POINTS,
+        "extras_require": {
+            "urwid": [
+                "urwid==1.3.1"
```

Comment:
Please don't specify a version number. It should work with any version :-)

Reply:
Until #34 is fixed would `~=1.3` be appropriate?

Reply:
No, I don't want to require any specific version. Hachoir has bugs, that's life :-)

Reply:
I mean. This PR doesn't make Hachoir worse.

But I don't want to use list of of requirements as a workaround for bugs.

Reply:
Sorry. I mean adding an optional dependency to urwid (any version) doesn't make Python worse. It just helps a few more people to use Hachoir.